projects/bird-atlas/src/app/+grid/grid-index/grid-index-map/grid-index-map.component.ts

```diff
@@ -9,6 +9,13 @@ import { BehaviorSubject, Subject } from 'rxjs';
 import { filter, takeUntil } from 'rxjs/operators';
 import { AtlasGrid, AtlasGridSquare } from '../../../core/atlas-api.service';
 
+type AtlasActivityCategory =
```

Comment:
This type definition could be on the `atlas-api.service` where the other types are (so then the variable `activityCategory` of the `AtlasGridSquare` interface would be like this `activityCategory?: {key: AtlasActivityCategory; value: string}`)